include/uxr/client/core/session/stream/stream_id.h

```diff
@@ -44,7 +44,9 @@ typedef enum uxrStreamType
     /** Identifies a best-effort stream. */
     UXR_BEST_EFFORT_STREAM,
     /** Identifies a reliable stream. */
-    UXR_RELIABLE_STREAM
+    UXR_RELIABLE_STREAM,
+    /** Identifies a brokerless stream. */
+    UXR_BROKERLESS_STREAM
```

Comment:
This is the stream name, in the other file `UXR_BROKERLESS_STREAM` is not being used.

Reply:
Everything is ok. My fault.